pom.xml

```diff
@@ -75,16 +87,12 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.8.0</version>
```

Comment:
Why are you removing the version?

Reply:
It's copied from azure: https://github.com/IABTechLab/uid2-attestation-azure/blob/master/pom.xml#L96

Weird we didn't specify a version there.

I will add them back.

Reply:
If not specified, `mvn help:effective-pom` shows it will use `3.1` version.
Add back `<version>3.8.0</version>`